.github/workflows/ci.yml

```diff
@@ -243,17 +243,38 @@ jobs:
         log-level: error
         arguments: --all-features --target ${{ matrix.platform.target }}
 
+  eslint:
+    name: ESLint
+
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./src/platform_impl/web/script
+
+    steps:
+      - name: Checkout
+        uses: taiki-e/checkout-action@v1
+      - name: Setup NPM
+        run: npm install
+      - name: Run ESLint
+        run: npx eslint
+
   swc:
     name: Minimize JavaScript
+
     runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./src/platform_impl/web/script
 
     steps:
-      - uses: taiki-e/checkout-action@v1
+      - name: Checkout
+        uses: taiki-e/checkout-action@v1
```

Comment:
No reason to add a name to this action

Reply:
I actually think all actions should have names to make it easier read the CI.
But it would be inconsistent with our current setup!

Done!